code/modules/awaymissions/maploader/reader.dm

```diff
@@ -316,7 +323,7 @@ var/list/map_dimension_cache = list()
 	//instanciate the last /turf
 	var/turf/T = instance_atom(members[last_turf_index],members_attributes[last_turf_index],xcrd,ycrd,zcrd,rotate)
 
-	if(first_turf_index != last_turf_index) //More than one turf is present - go from the lowest turf to the turf before the last one
+	if(T && first_turf_index != last_turf_index) //More than one turf is present - go from the lowest turf to the turf before the last one
```

Comment:
I don't know if you're messing with me on purpose or by accident.
Under what circumstances can T be null here?

Reply:
Testing it again and the runtime doesn't seem to be showing up anymore, so removed this.